docs/content/Caching/Running-in-Production.md

```diff
@@ -0,0 +1,212 @@
+---
+title: Running in Production
+permalink: /caching/running-in-production
+category: Caching
+menuOrder: 4
+---
+
+Cube.js makes use of two different kinds of cache:
+
+- Redis, for in-memory storage of query results
+- Cube Store for persisting pre-aggregations
+
+Redis is included by default in development mode. In production, we **strongly**
```

Comment:
Redis isn't included in the development mode. In the development mode, the queue and cache are done in memory.